hangman.py

```diff
@@ -200,14 +255,14 @@ def hangman_with_hints(secret_word):
 
     # To test part 2, comment out the pass line above and
     # uncomment the following two lines.
```

Comment:
I think you can remove it as we don't need these comments in future

Reply:
done